src/activities/Elsa.Activities.BlobStorage/Activities/DeleteBlob.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Elsa.ActivityResults;
+using Elsa.Attributes;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Storage.Net.Blobs;
+
+// ReSharper disable once CheckNamespace
+namespace Elsa.Activities.BlobStorage
+{
+    [Action(
+        Category = "BlobStorage",
+        Outcomes = new[] { OutcomeNames.Done }
+    )]
+    public class DeleteBlob : Activity
+    {
+        public DeleteBlob(IBlobStorage storage)
+        {
+            _storage = storage;
+        }
+        private readonly IBlobStorage _storage;
+
+        [ActivityProperty(Hint = "The Ids of the blobs", 
```

Comment:
Although we use `Id` when naming variables, when writing plain English descriptions, comments etc. we should use "ID". Using `Id` is just a C# naming convention where we use _PascalCasing_ for abbreviations. Unless it concerns two-letter acronyms such as IO (Input Output). Oftentimes, people confuse ID to be an acronym, although it's not: it's simply an abbreviation of "Identity", which is why the naming convention we use dictates it to be Id instead of ID. But when it comes to regular English, different rules apply.